js/lib/ui.js

```diff
@@ -304,7 +304,9 @@ elgg.ui.initDatePicker = function() {
 				}
 			},
 			nextText: '&#xBB;',
-			prevText: '&#xAB;'
+			prevText: '&#xAB;',
+			changeMonth: true,
+			changeYear: true
```

Comment:
i know we have no support for older version of IE, but still this could potentially break a lot of sites